.github/dependabot.yml

```diff
@@ -0,0 +1,19 @@
+version: 2
+updates:
+- package-ecosystem: npm
+  directory: "/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  open-pull-requests-limit: 10
+  versioning-strategy: increase
+  rebase-strategy: disabled
+- package-ecosystem: npm
+  directory: "/playground/"
+  schedule:
+    interval: weekly
+    time: "04:00"
+  open-pull-requests-limit: 10
+  versioning-strategy: increase
+  rebase-strategy: disabled
+  
```

Comment:
New line at the end of line -> editorconfig :)

Reply:
This is weird because I see the newline. Or my eyes are playing tricks

Reply:
<img width="459" alt="Capture d’écran 2021-02-23 à 14 04 04" src="https://user-images.githubusercontent.com/20380692/108847467-03b13000-75e0-11eb-90fc-e81a92369a11.png">

This symbol means you have don't have a new line at the end of your file.

Reply:
Here you have an empty line + the end of the file, but this is not a newline at the end